client/js/nonprofits/donate/info-step.js

```diff
@@ -38,8 +37,14 @@ function init(donation$, parentState) {
   state.showDedicationForm$ = flyd.map(()=> false, state.submitDedication$)
 
   // Save donor supporter record
-  state.supporterFields = supporterFields.init({required: {email: true}}, parentState.params$)
-  state.savedSupp$ = flyd.flatMap(postSupporter , flyd.map(formatFormData, state.submitSupporter$))
+  const formData$ = flyd.map(formatFormData, state.submitSupporter$)
+
+  state.supporterFields = supporterFields.init({required: {email: true}}, state.params$)
+  state.savedSupp$ = flyd.flatMap(postSupporter, flyd.map(
+    supporter => ({ ...supporter, tags: state.params$().tags }),
+    formData$
+  ))
+
```

Comment:
Have you tested that this actually works? Flimflam code gives me a headache :smile:

Reply:
Yes, it works haha It was a headache to make it work, I have to admit.

Reply:
What happens if no tags are provided? Does this still work?

Reply:
Yes. I tested with an empty string, replacing `state.params$().tags` with undefined and if the `data-tags` attribute is not passed on the HTML.